tests/Behat/Context/FeatureContext.php

```diff
@@ -282,4 +363,30 @@ protected function getCaretButton($block)
 
         return $button;
     }
+
+    /**
+     * @param $block
+     * @param $name
+     * @return mixed
+     */
+    protected function findFieldInBlock($block, $name)
+    {
+        $label = $block->findAll('xpath', sprintf('//label[contains(text(), \'%s\')]', $name));
+
+        assertNotNull($label, sprintf('Could not find a label for a field with the content "%s"', $name));
```

Comment:
In the case of `Title` I get the following issue with this:
`Found more than one label containing the phrase "Title".
      Failed asserting that actual size 0 matches expected size 1.`

The HTML looks as follows: 
`<label for="Form_ElementForm_11_PageElements_11_Title" class="form__field-label ">Title (displayed if checked)</label>`
and in the feature file I've got 
`And I fill in "Charlie's Block" for "Title" for block 2`

Reply:
Okay, I found the mistake I made. This works as expected, but the message is wrong. Should assert the count instead.